framework/math/quadratures/angular/product_quadrature.cc

```diff
@@ -165,8 +163,8 @@ AngularQuadratureProdGL::AngularQuadratureProdGL(int Nphemi, bool verbose) : Pro
 
 AngularQuadratureProdGLL::AngularQuadratureProdGLL(int Na, int Np, bool verbose)
```

Comment:
what is the purpose of a product Gauss Legendre Legendre quad? The only one that matters is a Gauss Legendre Chebyshev.

Reply:
I'll add a note to remove it!